docs/source/algorithms.rst

```diff
@@ -50,3 +50,85 @@ Using the same example file as before, we can execute a GaussFilter on our test
     gf.filterExperiment(exp)
     # MzMLFile().store("test.filtered.mzML", exp)
 
+
+
+Spectra Merge Algorithm
+*************************
+
+The pyOpenMS spectra merge algorithms allows the merge of several spectra by either increasing S/N ratio (for MS1 and above) or merging scans which stem from similar precursors (for MS2 and above). In any case, the number of scans will be reduced. In addition, pyOpenMS provides the method to average spectra. 
+
+Different spectra merge algorithms are available in pyOpenMS:
+
+- mergeSpectraBlockWise
+- mergeSpectraPrecursors
+- average (over neighbouring spectra for MS1 or above)
+
+In order to have a better picture of the algorithms, each one will be introduced by an example below. We will use another example file compared to above and start with an example of merging spectra blockwise:
+
+.. code-block:: python
+
+    # retrieve data 
+    gh = "https://raw.githubusercontent.com/OpenMS/pyopenms-extra/master"
+    urlretrieve(gh + "/src/data/small.mzML", "test.mzML")
+
+    # load MS data and store as MSExperiment object
+    exp = MSExperiment()
+    MzMLFile().load('test.mzML', exp)
+
+    spectra = exp.getSpectra()
+    print(f"There are {len(spectra)} spectra before merging.\n")
+
+    # merges blocks of MS or MS2 spectra
+    sm = SpectraMerger()
+    sm.mergeSpectraBlockWise(exp)
+
+    spectraMB = exp.getSpectra()
+    print(f"There are {len(spectraMB)} spectra after merging")
+
+
+.. code-block:: output 
+  
+    There are 236 spectra before merging.
+
+    There are 90 spectra after merging. 
+
+
+As predicated, the number of scans decreased after merging MS spectra. The modified data structure can be stored on disk:
+
+.. code-block:: python
+
+    MzMLFile().store("mergedBlockWise.mzML", exp)
+
+
+The merging of spectra with similar precursors progresses likewise:
+
+.. code-block:: python 
+
+    # load MS data and store as MSExperiment object
+    exp = MSExperiment()
+    MzMLFile().load('test.mzML', exp)
+
+    # merge spectra with similar precursors 
```

Comment:
How is similar defined?

Reply:
Similar to the processing steps, I think that I should add a more detailed information block in each example for better understanding.

Reply:
I think he means when are pcs considered similar (m/z ? Charge?)